tutorials/Example_Track_on_Radar_Segment_on_Satellite/track_on_radar_segment_on_satellite.ipynb

```diff
@@ -0,0 +1,1245 @@
+    }
+   ],
+   "source": [
+    "goes_data['time_bounds'][0].values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "numpy.datetime64('2021-05-26T15:56:15.477525888')"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "goes_data['time_bounds'][0].values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/bj/m6g82c6n41g83y3_dx02y7ch0000gp/T/ipykernel_29580/584919884.py:35: RuntimeWarning: invalid value encountered in sqrt\n",
+      "  r_s = (-1.0 * b_var - np.sqrt((b_var**2) - (4.0 * a_var * c_var))) / (2.0 * a_var)\n"
+     ]
+    }
+   ],
+   "source": [
+    "llons, llats  = lat_lon_reproj(goes_data)\n",
+    "full_goes_data = goes_data['CMI_C10'].values\n",
+    "in_goes_for_tobac = xr.Dataset(data_vars=\n",
+    "    {\n",
+    "        \"C10\": ((\"time\",\"Y\", \"X\"), [full_goes_data],)\n",
+    "    },\n",
+    "    coords=\n",
+    "    {\n",
+    "        \"time\": [goes_data['time_bounds'][0].values],\n",
+    "    \"longitude\": ([\"Y\", \"X\"], llons),\n",
+    "    \"latitude\": ([\"Y\", \"X\"], llats)},\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tobac\n",
+    "import tobac.utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_detection_params = dict()\n",
+    "feature_detection_params['threshold'] = [30, 40, 50]\n",
+    "feature_detection_params['target'] = 'maximum'\n",
+    "feature_detection_params['position_threshold'] = 'weighted_diff'\n",
+    "feature_detection_params['n_erosion_threshold'] = 2\n",
+    "feature_detection_params['sigma_threshold'] = 1\n",
+    "feature_detection_params['n_min_threshold'] = 4\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xr_grid_full = xr_grid['reflectivity'][:,4]\n",
+    "xr_grid_full['z'] = xr_grid_full.z.assign_attrs({'standard_name': 'altitude'})\n",
+    "xr_grid_full['lat'] = xr_grid_full.lat.assign_attrs({'standard_name': 'latitude'})\n",
+    "xr_grid_full['lon'] = xr_grid_full.lon.assign_attrs({'standard_name': 'longitude'})\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid_iris = xr_grid_full.to_iris()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "out_fd = tobac.feature_detection.feature_detection_multithreshold(grid_iris, 0, **feature_detection_params)"
```

Comment:
We should rename this to be consistent with the other examples: `out_fd` -> `features`